scripts/test-unit.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
I think this file is not needed it's an old one and we don't use it anymore to detect what was changed as we use github built-in functionally

Reply:
I added it back because we still reference it in the makefile (`make test-unit`) and it is useful for running all the tests locally. If we have another way to run tests locally though then happy to replace it with that?